src/cachelib/__init__.py

```diff
@@ -16,3 +16,9 @@
     "UWSGICache",
 ]
 __version__ = "0.2.0"
+
+
+import logging
+from logging import NullHandler
+
+logging.getLogger(__name__).addHandler(NullHandler())
```

Comment:
Shouldn't need to add a null handler. My understanding is that libraries should do no logging configuration, only emit messages.

Reply:
@davidism The idea was preventing events from being outputted to `sys.stderr` in the absence of logging configuration as pointed out in the [docs](https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library). I'm OK with it If you think the default behaviour is desirable. Should we just remove this then?